activerecord/lib/active_record/connection_adapters/abstract/schema_definitions.rb

```diff
@@ -680,9 +680,10 @@ def remove_references(*args, **options)
       end
       alias :remove_belongs_to :remove_references
 
-      # Adds a foreign key.
+      # Adds a foreign key from the receiver to the supplied table name.
```

Comment:
I don't see `receiver` used in any other method docs for this class. Maybe "Adds a foreign key _to the table using a supplied table name_" would sound better?